File: src/main/scala/de/theunknownxy/mcdocs/ClientProxy.scala
package de.theunknownxy.mcdocs

import net.minecraft.client.settings.KeyBinding
import org.lwjgl.input.Keyboard
import cpw.mods.fml.client.registry.ClientRegistry
import cpw.mods.fml.common.FMLCommonHandler

class ClientProxy extends CommonProxy {
  
  override def setupKeys() {
     ClientRegistry.registerKeyBinding(KeyBindings.openDocBinding)
     FMLCommonHandler.instance().bus().register(new KeyInputHandler())
  }
}

File: src/main/kotlin/de/theunknownxy/mcdocs/ClientProxy.kt
package de.theunknownxy.mcdocs

import cpw.mods.fml.client.registry.ClientRegistry
import cpw.mods.fml.common.FMLCommonHandler

public class ClientProxy : CommonProxy() {
    override fun setupKeys() {
        ClientRegistry.registerKeyBinding(KeyBindings.openDocBinding)
        var keyhandler = KeyInputHandler()
        keyhandler.init()
        FMLCommonHandler.instance().bus().register(keyhandler)
    }
}